### ISSUE TITLE
When tagging people, many duplicates of the same person appear

### ISSUE THREAD

**Issue by urosino** [RocketChat/Rocket.Chat#20874]
### Description:
When you want to tag people, using @ same person appears multiple times.
Also, their online status is not correct.

### Steps to reproduce:
1. Go in to some (private) room
2. Hit @ and start typing initials 
3. The same person appears multiple times

### Expected behavior:
1. To appear only once and online status to be accurate

### Actual behavior:
This is how the information is displayed.
![qslush0v](https://user-images.githubusercontent.com/25570106/108826007-1158bc80-75c4-11eb-80ee-ded2066add7d.gif)

### Server Setup Information:
* Version of Rocket.Chat Server: 3.11.1
* Operating System:  Docker image
* Deployment Method: Azure Kubernetes Pod
* Number of Running Instances: 1
* DB Replicaset Oplog: rs0
* NodeJS Version: v12.18.4
* MongoDB Version: 4.0.12 / mmapv1

### Client Setup Information
- Desktop App or Browser Version: Desktop App, 3.1.1

- Operating System: Windows 10

### Additional context
 - No additional context

### Relevant logs:
 - No other logs

**Comment by lequy1010** [comment]
it's fixed here 
https://github.com/RocketChat/Rocket.Chat/pull/20484

**Comment by KevLehman** [comment]
Duplicate of #20483
